samples/waitForOperation.js

```diff
@@ -0,0 +1,49 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Waits for an operation to be completed. Calling this function will block until the operation is finished.
+ * @param {string} projectId - ID or number of the project you want to use.
+ * @param {Operation} operation - Operation instance you want to wait.
+ */
+function main(projectId, operation) {
```

Comment:
I am not sure how to test this file, cause we need to pass operation object from CLI params

Reply:
Just pass the params needed to create an `operation` object and create it inside `main`.